src/assets/js/languageSelector.js

```diff
@@ -1,96 +1,91 @@
-$(document).ready(function() {
-  languageSelector.init();
-});
+var languageSelector = {};
 
-var excludedSDK = ["android", "php"];
+languageSelector.excludedSDK = ['android', 'php'];
+languageSelector.currentLanguage = document.location.pathname.split('/')[2];
 
-var languageSelector = {
-  init: function() {
-    if ($('#language-selector').length === 0) {
-      return;
-    }
+languageSelector.init = function (select) {
+  this.setSelectorOptions(select);
 
-    var
-      self = this,
-      languageParam = document.location.pathname.split('/')[2];
+  select.select2({
+    minimumResultsForSearch: -1,
+    theme: 'material',
+    templateResult: this.setTemplate,
+    templateSelection: this.setTemplate
+  });
 
-    this.setSelectorOptions();
-
-    $('#language-selector [value="' + languageParam + '"]').attr(
-      'selected',
-      true
-    );
-
-    $('#language-selector').select2({
-      minimumResultsForSearch: -1,
-      theme: 'material',
-      templateResult: this.selectTemplate,
-      templateSelection: this.selectTemplate
-    });
-
-    $('.select2-selection__arrow')
-      .addClass('material-icons')
-      .html('arrow_drop_down');
+  $('.select2-selection__arrow')
+    .addClass('material-icons')
+    .html('arrow_drop_down');
+    
+  this.setCurrentLanguage(select);
+  this.onChange(select);
+};
 
-    $('#language-selector').on('change', function() {
-      var language = $(this).val();
-      window.location.assign(self.getLatestVersionURL(language));
-    });
-  },
+languageSelector.setCurrentLanguage = function (select) {
+  select
+    .val(this.currentLanguage)
+    .change();
+};
 
-  getLatestVersionURL: function(language) {
-    if (language === "javascript") {
-      language = "js";
-    }
+languageSelector.onChange = function (select) {
+  var self = this;
+  select.on('change', function() {
+    var language = $(this).val();
+    window.location.assign(self.getLatestVersionURL(language));
+  });
+};
 
-    var
-      baseUrl = window.location.protocol + '//' + window.location.host,
-      latestVersion = '';
+languageSelector.getLatestVersionURL = function (language) {
+  var
+    baseUrl = window.location.protocol + '//' + window.location.host,
+    latestVersion = '',
+    customPathname = '';
+  
+  if (language === 'javascript') {
+    language = 'js';
+  }
+  
+  latestVersion = Object.keys(sdkVersions[language]).sort().slice(-1)[0];
+  customPathname = '/sdk-reference/' + language + '/' + latestVersion + '/essentials';
+  
+  return baseUrl + customPathname;
+};
 
-    Object.keys(sdkVersions[language]).forEach(function(key) {
-      if (sdkVersions[language][key] === "master") {
-        latestVersion = key;
+languageSelector.setSelectorOptions = function (select) {
+  var self = this;
+  Object.keys(sdkVersions)
+    .filter(function(v) { return !self.excludedSDK.includes(v); })
```

Comment:
`Array.includes` is an ES6 feature.

Reply:
It seems that all major browser support this feature now: https://caniuse.com/#feat=array-includes

Reply:
That's besides the point. This file is written exclusively using ES5. I don't know the reasons, but if we allow a single ES6 feature, then I have many more comments to add to get rid of outdated syntax.